Composer/packages/client/src/pages/publish/publishPageUtils.tsx

```diff
@@ -34,13 +34,8 @@ export const generateBotPropertyData = (botProjectData: BotProjectType[]) => {
 const findSkillManifestUrl = (skillManifests: SkillManifestFile[], hostname: string, appId: string): string[] => {
   const urls: string[] = [];
   for (const skillManifest of skillManifests || []) {
```

Comment:
Why do u need the conditional \[\]. Its typed to be an array

Reply:
have removed the condition, we just need one url.